yb_migrate/cmd/root.go

```diff
@@ -72,6 +74,9 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVarP(&exportDir, "export-dir", "e", "",
 		"export directory (default is current working directory")
+
+	rootCmd.PersistentFlags().BoolVarP(&askPromptResp, "assumeyes", "y", false,
```

Comment:
Name the command as `--yes`.
Rename the variable `askPromptResp` as `doNotPrompt` (note that the meaning is negated.)

Reply:
Will it be possible to populate the option value directly in the `&utils.DoNotPrompt` ? With that you won't need the `initCommonGlobalVars()` section.